src/hooks/useCourtContracts.js

```diff
@@ -72,7 +73,7 @@ export function useANJActions() {
   const activateANJ = useCallback(
     amount => {
       return jurorRegistryContract.activate(amount, {
-        gasLimit: ANJ_ACTIONS_GAS_LIMIT,
+        gasLimit: ANJ_ACTIVATE_GAS_LIMIT,
```

Comment:
I think we should have also use this limit for `jurorRegistryContract.approveAndCall` ? Which does staking and activating

Reply:
Yes, good catch!